### ISSUE TITLE
When on beat saber 1.31?

### ISSUE THREAD

**Issue by Molwi** [ToniMacaroni/SaberFactory#164]
When saber factory is going to support beat saber 1.31????? It was more than one month or idk is it abandoned??

**Comment by lividhen** [comment]
Dev seems to currently be working on RedLoader, but they will most likely update SaberFactory when they have the time.

**Comment by Alyng0** [comment]
Whats RedLoader

**Comment by Majorkrabbe** [comment]
It was better when you just had to put the customsaber.dll into the plugin folder. It did not have to be updated all the time....

**Comment by ghost** [comment]
bro that also had to be updated everytime beat saber did some changes deeper in the code

**Comment by Majorkrabbe** [comment]
That's not true. Before, I could use the same .dll file when beat saber was updated, I just copy and paste it.

**Comment by ghost** [comment]
You likely never had to update because you started using SaberFactory within a period where new Beat Saber updates didn’t introduce breaking changes to BSIPA (The code injection tool Beat Saber modders use)

The most recent updates for Beat Saber changed the way certain things need to be done in the mod, and Toni hasn’t gotten around to implementing that, on top of the changes he’s been making to revamp SaberFactory in v3.

These breaking changes can happen at any time, mainly when a new major update is introduced to Beat Saber, or they update the tools they’re using.

Not too long ago, the Beat Saber devs updated the version of Unity they’re using. That’s probably what caused a required update for SaberFactory.
